.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
 - git config --global url.git@github.com:.insteadof https://github.com/
 
 install:
-- make dep
+- make mod-tidy
```

Comment:
this step is installs dependencies but maybe the script should be `make lint test`, then the it has the same effect and the install step can be removed. We still pull all the dependencies but get extra linting like we also do in other repos

Reply:
`mod-tidy` *modifies* `go.mod` file. It's really bad idea to make this on CI. Latest build system doesn't require install any dependencies. Just `make lint` and `make race-nocache` will be enough.